### PR TITLE
[Azure Stack] Enable swagger to sdk for multi-api version

### PR DESCRIPTION
**Pull request by vishrutshah**
Fully synced from azure-stack/config.json

Regarding https://github.com/Azure/azure-sdk-for-ruby/issues/904

Checklist:

- [x] Include all the azure_mgmt_* from `config.json`
- [x] Remove `debug`
- [x] Remvoe `verbose`
- [x] Remove `use`
- [x] Make sure the Multi-api `AutoRest.Ruby` generator has been released and that is being used here
- [x] Update all .gemspec to ignore `build.json` file and that'd not be shipped while bundling
- [x] Make sure not to touch profiles

### CODE REVIEW

File: swagger_to_sdk_config.json [hunk 1]
```diff
@@ -5,494 +5,1501 @@
...
+        "package-name": "azure_mgmt_datalake_store",
+        "tag": "package-2015-10-preview"
       },
-      "output_dir": "management/azure_mgmt_devtestlabs/lib",
-      "wrapper_filesOrDirs": [
-        "azure_mgmt_devtestlabs.rb"
-      ]
+      "output_dir": "management/azure_mgmt_datalake_store/lib/2015-10-01-preview",
+      "generated_relative_base_directory": "2015-10-01-preview",
+      "build_dir": "management/azure_mgmt_datalake_store/lib/2015-10-01-preview"
```
Comment (veronicagg) [inline swagger_to_sdk_config.json]: are we missing devtestlabs?

File: swagger_to_sdk_config.json [hunk 2]
```diff
@@ -5,494 +5,1501 @@
...
+        "package-version": "0.11.0",
         "package-name": "azure_mgmt_recovery_services_backup",
-        "package-version": "0.11.0"
+        "tag": "package-2016-06"
+      },
+      "output_dir": "management/azure_mgmt_recovery_services_backup/lib/2016-06-01",
+      "generated_relative_base_directory": "2016-06-01",
+      "build_dir": "management/azure_mgmt_recovery_services_backup/lib/2016-06-01"
+    },
+    "azure_mgmt_recovery_services_2016_08_10": {
```
Comment (veronicagg) [inline swagger_to_sdk_config.json]: should we call this recovery_services_site_recovery? I think we should update it in the config.json, thoughts?

File: swagger_to_sdk_config.json [hunk 3]
```diff
@@ -5,494 +5,1501 @@
...
     }
   },
   "projects": {
-    "azure_mgmt_analysis_services": {
+    "azure_mgmt_analysis_services_2017_07_14": {
       "markdown": "specification/analysisservices/resource-manager/readme.md",
       "autorest_options": {
-        "namespace": "Azure::ARM::AnalysisServices",
+        "namespace": "Azure::ARM::AnalysisServices::Api_2017_07_14",
+        "package-version": "0.11.0",
```
[the remark this answers is not in the document]
Comment (vishrutshah) [reply swagger_to_sdk_config.json]: Yup. must be `0.14.0`

File: swagger_to_sdk_config.json (hunk 1 above)
Comment (vishrutshah) [reply swagger_to_sdk_config.json]: Aha ... may be let me check

File: swagger_to_sdk_config.json (hunk 2 above)
Comment (vishrutshah) [reply swagger_to_sdk_config.json]: good idea

**Comment by vishrutshah** [comment]
@veronicagg & I were discussing what should be the AutoRest.Ruby version in the swagger_to_sdk file. Based on the offline discussion we'll keep latest and promote multi-api version changes to latest,.

**Comment by vishrutshah** [comment]
Thanks for the review!!

**Pull request merged**